src/masonite/orm/grammar/BaseGrammar.py

```diff
@@ -93,6 +94,7 @@ def _compile_alter(self):
                 if column.length
                 else "",
                 nullable="" if column.is_null else " NOT NULL",
+                after=" AFTER " + "`{0}`".format(column.after_column) if column.after_column else ""
```

Comment:
@josephmancuso what could be a better way to write it?

Reply:
you have to put this string inside an `after_column_string()` method on the `MySQLGrammer` class and then change this line to 

```
after=self.after_column_string().format(after=column.after_column) if column.after_column else ""
```

On the `MySQLGrammer` class create a method like this:

```python
def after_column_string(self):
    return "AFTER {after}"
```